packages/react-reconciler/src/__tests__/ReactSuspenseEffectsSemantics-test.js

```diff
@@ -1980,11 +1980,10 @@ describe('ReactSuspenseEffectsSemantics', () => {
       });
 
       // @gate enableLegacyCache
-      // @gate replayFailedUnitOfWorkWithInvokeGuardedCallback
       it('are properly handled for layout effect creation', async () => {
         let useLayoutEffectShouldThrow = false;
 
-        function ThrowsInLayoutEffect() {
+        function ThrowsInLayoutEffect({unused}) {
```

Comment:
These tests used to only run in dev.

In prod we don't disable logs during component stack generation, so when the component stack gets generated it ends up logging again. We can avoid that by ensure component stacks error before the log. E.g. by destructuring props.